config/job.js

```diff
@@ -64,9 +64,11 @@ const SCHEMA_STEPS = Joi.array().items(SCHEMA_STEP).min(1);
 const SCHEMA_IMAGE = Joi.string();
 const SCHEMA_SETTINGS = Joi.object().optional();
 const SCHEMA_TEMPLATE = Joi.string().regex(Regex.FULL_TEMPLATE_NAME);
+const SCHEMA_DESCRIPTION = Joi.string().max(100).optional();
```

Comment:
Right now a lot of them are not alphabetically sorted so should I just move `SCHEMA_DESCRIPTION` above `SCHEMA_IMAGE` or sort them all?